pytket/docs/extensions_index.rst

```diff
@@ -0,0 +1,165 @@
+
+The additional modules can be installed adding the extension name to the installation command for pytket. For example pytket-quantinuum can be installed by running
+
+::
+
+   pip install pytket-quantinuum
+
+The types of ``Backend`` available in pytket are the following
+
+Types of Backend
+================
+
+* **QPUs** - These are real quantum computers that return shots based results. E.g the `QuantinuumBackend <https://tket.quantinuum.com/extensions/pytket-quantinuum/api.html#pytket.extensions.quantinuum.QuantinuumBackend>`_ .
+* **Cloud Access** - Cloud backends allow pytket to interface with cloud platforms to access additional QPUs and simulators. E.g `BraketBackend <https://tket.quantinuum.com/extensions/pytket-braket/api.html#pytket.extensions.braket.BraketBackend>`_ .
+* **Emulators** - These classically simulate a circuit and produce shots based results. Sometimes emulators use a noise model and have connectivity constraints to emulate real QPUs. E.g. `IBMQEmulatorBackend <https://tket.quantinuum.com/extensions/pytket-qiskit/api/api.html#pytket.extensions.qiskit.IBMQEmulatorBackend>`_ .
+* **Statevector Simulators** - Calculates the pure quantum state prepared by a circuit returning a vector/ndarray. Examples of statevector simulators are the `ForestStateBackend <https://tket.quantinuum.com/extensions/pytket-pyquil/api/api.html#pytket.extensions.pyquil.ForestStateBackend>`_ and the `AerStateBackend <https://tket.quantinuum.com/extensions/pytket-qiskit/api/api.html#pytket.extensions.qiskit.AerStateBackend>`_ . 
+* **Unitary Simulators** - Unitary simulators calculate the unitary operator that is applied by a circuit. A unitary matrix/ndarray is returned `AerUnitaryBackend <https://tket.quantinuum.com/extensions/pytket-qiskit/api/api.html#pytket.extensions.qiskit.AerUnitaryBackend>`_ is an example of such a simulator.
+* **Density Matrix Simulators** - These simulators compute the density matrix prepared by a circuit. The result can be a statistical mixture of states in contrast to statevector simulation. E.g. `CirqDensityMatrixSampleBackend <https://tket.quantinuum.com/extensions/pytket-cirq/api/api.html#pytket.extensions.cirq.CirqDensityMatrixSampleBackend>`_ .
+* **Other specialised simulators** - There are extensions for simulating specific types of circuit. For instance the `SimplexBackend <https://tket.quantinuum.com/extensions/pytket-pysimplex/api/pysimplex/api.html#pytket.extensions.pysimplex.SimplexBackend>`_ is designed to simulate Clifford circuits. 
+
+A full list of available pytket backends is shown below.
+
+QPUs
+====
+`IBMQBackend <https://tket.quantinuum.com/extensions/pytket-qiskit/api.html#pytket.extensions.qiskit.IBMQBackend>`_
```

Comment:
In which order are the QPUs sorted here?

Reply:
![Screenshot 2023-10-31 at 18 55 00](https://github.com/CQCL/tket/assets/93673602/2785cf91-4a69-4224-94fd-a3904add66a0)

They are now in this order. I have now put Quantinuum first and mentioned that H2-1 is supported and H1-2 is not. Is this correct?

Reply:
I thought they are all three now available? I think we should list all three

Reply:
H1-2 is retired from my understanding.

Is there another one besides H1-1 and H2-1?

Reply:
I think the H1-2 can be left out.